src/modules/attitude_estimator_q/attitude_estimator_q_params.c

```diff
@@ -94,6 +103,16 @@ PARAM_DEFINE_FLOAT(ATT_MAG_DECL, 0.0f);
 PARAM_DEFINE_INT32(ATT_MAG_DECL_A, 1);
 
 /**
+ * External heading usage mode (from Mocap/Vision)
+ * Set to 1 to just use external yaw.
+ *
+ * @group Attitude Q estimator
+ * @min 0
+ * @max 1
+ */
+PARAM_DEFINE_INT32(ATT_EXT_HDG_M, 0);
```

Comment:
@mhkabir Could you please introduce 1 and 2 here so people can select between vision and mocap? We have quite a few having both active at the same time (e.g. for ground truth) and want to be sure they can pick which source it uses.

Thanks!